.github/workflows/release.yml

```diff
@@ -36,8 +36,6 @@ jobs:
       name: Run commands
       id: runcmd
       with:
-        arch: aarch64
```

Comment:
`uraimo/run-on-arch-action` is the action that runs a job on qemu. we do not need it now, so you should make it a regular `run` job, imho.